src/array/binary/builder.rs

```diff
@@ -129,6 +129,21 @@ impl<O: OffsetSizeTrait> WKBBuilder<O> {
         write_multi_polygon_as_wkb(&mut buf, &geom).unwrap();
         self.0.append_value(&buf)
     }
+
+    /// Push a geo::Geometry onto the end of this array
+    pub fn push_geo_geometry(&mut self, geometry: geo::Geometry) {
```

Comment:
Can you implement this in terms of `GeometryTrait` instead? Since `geo::Geometry` implements `GeometryTrait`, that's more general. You can see `MixedGeometryBuilder` for an example